content/en/docs/guides/setup/rpi4_setup/index.md

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
This is better documentation than that in #6. Let's keep this one :tada:

Reply:
Solved